frontend/src/layouts/DefaultLayout.vue

```diff
@@ -1,64 +1,54 @@
 <template>
-  <v-app style="background: #90A4AE">
-    <!-- left side drawer (desktop)-->
-    <v-navigation-drawer v-if="$vuetify.breakpoint.smAndUp"
-                         v-model="drawer" app
-                         clipped permanent
-                         :mini-variant.sync="mini"
-                         mini-variant-width="40"
-                         color="grey lighten-2">
-      <v-btn v-if="mini" icon>
-        <v-icon>mdi-format-list-bulleted-triangle</v-icon>
-      </v-btn>
-      <v-spacer />
-      <v-btn v-if="!mini" icon
-             fixed
-             class="ma-2"
-             style="z-index: 10;"
-             right @click.stop="mini = !mini">
-        <v-icon>mdi-chevron-left</v-icon>
-      </v-btn>
-
-      <v-divider />
-
-      <router-view v-if="!mini" name="aside" />
-    </v-navigation-drawer>
-
+  <v-app>
     <!-- second menu/tool bar -->
     <v-app-bar v-if="$vuetify.breakpoint.smAndUp"
                app
                clipped-left hide-on-scroll
                color="blue-grey darken-4" dark>
-      <v-toolbar-title class="pl-4 pr-4">
-        <i>🏕</i>️ eCamp
-      </v-toolbar-title>
       <v-toolbar-items>
-        <v-btn text
+        <v-btn text class="px-2"
                exact :to="{ name: 'home'}">
-          Home
-        </v-btn>
-        <v-btn text
-               :to="{ name: 'camps', params: { groupName: encodeURI('Pfadi Bewegung Schweiz') } }">
-          Camps
+          <v-toolbar-title>
+            <i>🏕</i>️<span class="ml-4 mr-2">eCamp<sup class="blue-grey--text">v3</sup></span>
+          </v-toolbar-title>
         </v-btn>
       </v-toolbar-items>
       <v-spacer />
-      <v-btn v-if="loggedIn" text
-             @click="logout">
-        <v-icon v-if="logoutIcon" :left="$vuetify.breakpoint.mdAndUp">{{ logoutIcon }}</v-icon>
-        <v-progress-circular v-else indeterminate
-                             size="18"
-                             class="mr-2" />
-        <span class="sr-only-sm-and-down">Log out</span>
-      </v-btn>
-      <v-btn v-else text
-             :to="{ name: 'login' }">
-        <v-icon :left="$vuetify.breakpoint.mdAndUp">mdi-login</v-icon>
-        <span class="sr-only-sm-and-down">Log in</span>
-      </v-btn>
-      <v-btn icon>
-        <v-icon>mdi-account</v-icon>
-      </v-btn>
+      <global-search />
+      <v-menu offset-y dark
```

Comment:
Da gibt es viel Redundanz zu `CampLayout`. Wenn wir mit 2 Layouts gehen, dann wär cool wenn wir gleichen Code in Komponenten auslagern so dass wir möglichst keinen kopierten Code haben.